templates/client-snapshot-agent-deployment.yaml

```
@@ -127,9 +127,9 @@ spec:
               readOnly: true
           {{- end }}
           {{- end }}
-      {{- if (or .Values.global.bootstrapACLs (and .Values.global.tls.enabled .Values.global.tls.enableAutoEncrypt)) }}
+      {{- if (or (or .Values.global.acls.manageSystemACLs .Values.global.bootstrapACLs) (and .Values.global.tls.enabled .Values.global.tls.enableAutoEncrypt)) }}
```

As you demonstrated elsewhere, it's possible to have three inputs to `or`.

```suggestion
      {{- if (or .Values.global.acls.manageSystemACLs .Values.global.bootstrapACLs (and .Values.global.tls.enabled .Values.global.tls.enableAutoEncrypt)) }}
```